pkg/cmd/pipelineresource/list.go

```diff
@@ -160,6 +160,8 @@ func details(pre v1alpha1.PipelineResource) string {
 	var key = "url"
 	if pre.Spec.Type == v1alpha1.PipelineResourceTypeStorage {
 		key = "location"
+	} else if pre.Spec.Type == v1alpha1.PipelineResourceTypeCloudEvent {
+		key = "targeturi"
```

Comment:
isn't the key case sensitive? `targetURI?` ?

Reply:
A few lines above the key is set to lower case before it's checked, so `targetURI` would not match.

Reply:
thank you!